From aca1f82501636f50e7ca7140d5c497e5006a4cbe Mon Sep 17 00:00:00 2001
From: Yuwei Chen <yuwei.chen@intel.com>
Date: Mon, 20 Jul 2020 14:48:46 +0800
Subject: [PATCH 1/1] Test case for changing include .h of .vfr file

Description: Change the define value in the MdePkg\Include\Uefi\UefiMultiPhase.h which included in the MdeModulePkg/Universal/DriverSampleDxe/Vfr.vfr file.

Repo: edk2

Signed-off-by: Yuwei Chen <yuwei.chen@intel.com>
---
 MdePkg/Include/Uefi/UefiMultiPhase.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/MdePkg/Include/Uefi/UefiMultiPhase.h b/MdePkg/Include/Uefi/UefiMultiPhase.h
index 50e4d700a62f..3087ced353ec 100644
--- a/MdePkg/Include/Uefi/UefiMultiPhase.h
+++ b/MdePkg/Include/Uefi/UefiMultiPhase.h
@@ -28,7 +28,7 @@ SPDX-License-Identifier: BSD-2-Clause-Patent
 ///
 /// NOTE: EFI_VARIABLE_AUTHENTICATED_WRITE_ACCESS is deprecated and should be considered reserved.
 ///
-#define EFI_VARIABLE_AUTHENTICATED_WRITE_ACCESS              0x00000010
+#define EFI_VARIABLE_AUTHENTICATED_WRITE_ACCESS              0x00000008
 
 #ifndef VFRCOMPILE
 #include <Guid/WinCertificate.h>
-- 
2.27.0.windows.1

